book/src/ledger-replication.md

```diff
@@ -31,7 +31,7 @@ core. The total space required for verification is `1_ledger_segment +
 Validators for PoRep are the same validators that are verifying transactions.
 They have some stake that they have put up as collateral that ensures that
 their work is honest. If you can prove that a validator verified a fake PoRep,
-then the validators stake can be slashed.
+then the validator will not receive a reward for that storage epoch.
```

Comment:
@garious, I'm a little confused about how this ties in with the vote accounts. A validator's stake usually lives in the vote account. Will the storage rewards need to be aware of the reward accounts for all delegated stakes? Just trying to understand how the two rewards fit together and whether they even need to.

Reply:
The stake only currently lives in the vote account. The Passive Delegated Stakes proposal moves them to a separate account so that the validator can send votes to just one account, no matter how many stakers delegate their stake to that validator.

Reply:
Okay so that should help simplify this a little. Maybe the storage account can carry the id that passive delegated account as well instead of us having to loop through all accounts associated with a node (whether via delegation or direct ownership). 

Our current plan is to setup a separate storage account for validators. Using a new keypair. In the current proposal these accounts don't carry any information about who their owner (validator) is. I think we'll need to add that though so we can correctly tie things together.  @sakridge thoughts ?

Reply:
What is the scenario where they need to?

Reply:
Current design is to not need slashing.

Reply:
For example, we'd need to co-related validator <-> storage_account to figure out how to deliver rewards

Reply:
The validator has the keypair for the storage account, it can use that to transfer the rewards anywhere it wants.